### PR TITLE
Kipod/feature/newsletter

### PR DESCRIPTION
**Pull request by kipod**
Newsletter backend

### CODE REVIEW

**Review by Taraslysak — APPROVED** [review]

File: tasks/create_dummy_data.py [hunk 1]
```diff
@@ -60,7 +58,9 @@ def create_footbal_coach():
         )
         db.add(football_coach)
         db.commit()
-        fc_sport = m.CoachSport(coach_id=football_coach.id, sport_id=SPORTS[0].id)
+        SPORT_ID: int = db.query(m.SportType).first().id
```
Comment (Taraslysak) [inline tasks/create_dummy_data.py]: why is it declared as const?

**Review by Taraslysak — COMMENTED** [review]
Please explain it to me

File: tasks/create_dummy_data.py (hunk 1 above)
Comment (kipod) [reply tasks/create_dummy_data.py]: will be fixed. thank u

**Review by BohdanSizomin — COMMENTED** [review]
Thank you
Now we know a little bit more about asyncio !

File: app/router/newsletter.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+from app.config import get_settings, Settings
+from app.logger import log
+from app.database import get_db
+import app.schema as s
+import app.model as m
+
+newsletter_router = APIRouter(prefix="/newsletter", tags=["Newsletter"])
+
+
+@newsletter_router.post("/")
```
Comment (varvarashcherbyna7) [inline app/router/newsletter.py:15]: I think it's better to add /subscription here, because we will have an unsubscribe later

**Review by varvarashcherbyna7 — COMMENTED** [review]
)

**Pull request merged**